packages/ui/src/components/ManageKafkaPermissions/components/ResourceType.tsx

```diff
@@ -72,7 +72,7 @@ export const ResourceType: React.VFC<ResourceTypeProps> = ({
         onToggle={onToggle}
         onSelect={onSelect}
         isOpen={isOpen}
-        width={200}
+        width={180}
```

Comment:
How is this hardcoded width working in regards to responsiveness?

Reply:
The stories are building, you can test the responsiveness here
https://63c984f3273772562ae72406-arwqkcroph.chromatic.com/?path=/story/components-managekafkapermissions-editpermissions--interactive-example-all-accounts-selected&globals=backgrounds.grid:true

Reply:
When this table was first implemented, we asked Sarah Rambacher to help out with the responsiveness of the table, but I'd have to dig up previous PRs that she worked on to see what the end result of her work was (and hope that she pasted screencaps there 😬). Looking at this storybook preview, I agree that it would be good to revisit the responsiveness.